drivers/ec_host_cmd_periph/Kconfig

```diff
@@ -35,6 +35,45 @@ config EC_HOST_CMD_PERIPH_ESPI
 	  Enable support for Embedded Controller host commands using
 	  the eSPI bus.
 
+config EC_HOST_CMD_PERIPH_SHI
+	bool "Host commands support using SHI"
+	help
+	  Enable support for Embedded Controller host commands using
+	  the Serial Host Interface.
+
 endchoice
 
+if EC_HOST_CMD_PERIPH_SHI
+
+choice
+	prompt "SHI driver"
+	default EC_HOST_CMD_PERIPH_SHI_NPCX if SOC_FAMILY_NPCX
+
```

Comment:
It is recommended that `choice` symbols are assigned a name: `choice EC_HOST_CMD_PERIPH_SHI_DRIVER`.

Reply:
Should the above choice for `prompt "Host commands peripheral"` also have a name?

Reply:
A separate small PR would be appropriate.

Reply:
Will do!